lms/templates/course.html

```diff
@@ -19,14 +23,18 @@ <h2 class="course-name">
         <span class="course-code">${course.display_number_with_default}</span>
         <span class="course-title">${course.display_name_with_default}</span>
       </h2>
-      <div class="course-date" aria-hidden="true">${_("Starts")}: ${course.start_datetime_text()}</div>
+##       <div class="course-date" aria-hidden="true">${_("Starts")}: ${course.start_datetime_text()}</div>
+      <div class="course-date localized_datetime" aria-hidden="true" data-format="shortDate" data-datetime="${date}" data-string="${container_string}"></div>
     </div>
     <div class="sr">
       <ul>
       <li>${course.display_org_with_default}</li>
       <li>${course.display_number_with_default}</li>
-      <li>${_("Starts")}: <time itemprop="startDate" datetime="${course.start_datetime_text()}">${course.start_datetime_text()}</time></li>
+      <li>${_("Starts")}: <time class="localized_datetime" itemprop="startDate" data-format="shortDate" data-datetime="${date}"></time></li>
```

Comment:
Why is the formatting of this start date different from the line above?  Here, the `Starts` is defined inline, whereas above it's in a `container_string`.  Can you make both use the "container_string" to make the code more readable?